src/types/user/interfaces.tsx

```diff
@@ -0,0 +1,11 @@
+import { UserData } from "./types";
```

Comment:
Does it makes sense to divide this interface into separate folder? Would u have a lot of interfaces in the app?